.github/workflows/canary-integration-test.yml

```diff
@@ -140,6 +140,7 @@ jobs:
 
     - name: deploy cluster
       run: |
+        yq write -d1 -i cluster/examples/kubernetes/ceph/operator.yaml "spec.template.spec.containers[0].image" rook/ceph:local-build
```

Comment:
Do these changes need to be made in a separate PR to enable this more broadly?

Reply:
Or shall we go with #8897?

Reply:
For now, I've had to add this test to my change. I just went ahead with #8897.